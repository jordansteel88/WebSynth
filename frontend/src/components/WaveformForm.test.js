// import React from "react";
// import { render } from "@testing-library/react";
// import WaveformForm from "./WaveformForm";

// it("renders without crashing", function() {
//   render(<WaveformForm />);
// });

// it("matches snapshot", function() {
//   const { asFragment } = render(<WaveformForm
//                                   controls={waveform} 
//                                   callback={setWaveform} />);
//   expect(asFragment()).toMatchSnapshot();
// });

// //props undefined