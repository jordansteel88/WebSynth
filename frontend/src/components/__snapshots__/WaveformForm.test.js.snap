// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="waveform"
  >
    <h4>
      Waveform
    </h4>
    <div
      class="WaveformForm-radios"
    >
      <input
        id="sine"
        name="waveform"
        type="radio"
        value="sine"
      />
      <label
        for="sine"
      >
        Sine
      </label>
      <input
        id="triangle"
        name="waveform"
        type="radio"
        value="triangle"
      />
      <label
        for="triangle"
      >
        Triangle
      </label>
      <input
        id="sawtooth"
        name="waveform"
        type="radio"
        value="sawtooth"
      />
      <label
        for="sawtooth"
      >
        Sawtooth
      </label>
      <input
        id="square"
        name="waveform"
        type="radio"
        value="square"
      />
      <label
        for="square"
      >
        Square
      </label>
    </div>
  </div>
</DocumentFragment>
`;
